packages/js/src/utils/id-token.test.ts

```diff
@@ -110,3 +116,172 @@ describe('decodeIdToken', () => {
     expect(() => decodeIdToken(jwt)).toThrowError(StructError);
   });
 });
+
+describe('verifyIdToken', () => {
+  test('valid ID Token should not throw', async () => {
+    const { privateKey, publicKey } = await generateKeyPair('RS256');
```

Comment:
we should also test `EC512` algorithm

Reply:
@IceHe `EC512` is recently introduced to our product, we should support it, thanks.

Reply:
Added `ES512`